src/main/java/io/debezium/connector/vitess/VitessConnectorConfig.java

```diff
@@ -68,6 +73,14 @@ public class VitessConnectorConfig extends RelationalDatabaseConnectorConfig {
             .withImportance(ConfigDef.Importance.HIGH)
             .withDescription("Password of the user to be used when connecting the Vitess VTGate gRPC server.");
 
+    public static final Field VTGATE_AUTH_FILE = Field.create(DATABASE_CONFIG_PREFIX + "auth_file")
+            .withDisplayName("Client Auth File")
+            .withType(Type.STRING)
+            .withWidth(Width.LONG)
+            .withImportance(ConfigDef.Importance.HIGH)
+            .withDescription("Client auth file to be used when connecting to the VTGate server. " +
+                    "Sample: https://github.com/vitessio/vitess/blob/main/examples/local/grpc_static_client_auth.json");
```

Comment:
Instead of linking an example here, I suggest skipping this here but explicitly providing an example in the documentation like this https://debezium.io/documentation/reference/stable/connectors/vitess.html#vitess-property-message-key-columns

The documentation source is in https://github.com/debezium/debezium/blob/main/documentation/modules/ROOT/pages/connectors/vitess.adoc

Reply:
The config table is nice. Consider adding a short text around https://debezium.io/documentation/reference/2.0/connectors/vitess.html#setting-up-vitess as well on the setup.